Resources/config/services.xml

```diff
@@ -65,18 +65,18 @@
         <!-- Listeners -->
         <service id="fsi_doctrine_extensions.listener.uploadable_post_load" class="%fsi_doctrine_extensions.listener.uploadable_post_load.class%">
             <argument type="service" id="fsi_doctrine_extensions.listener.uploadable" />
-            <tag name="fsi_doctrine_extensions.listener.uploadable" />
+            <tag name="fsi_doctrine_extensions.listener.uploadable" priority="2" />
```

Comment:
Can we place this priorities more widely? We can later put events in end application in between those events. Symfony has similar approach: http://symfony.com/doc/current/reference/events.html#kernel-request